docs/general/installation/container.md

```diff
@@ -311,28 +311,21 @@ SuccessExitStatus=0 143
 WantedBy=default.target
 ```
 
-## TrueNAS SCALE / TrueCharts
+## TrueNAS SCALE
 
-Jellyfin is available as a [TrueNAS SCALE](https://www.truenas.org/) App inside the [TrueCharts](https://www.truecharts.org/) App Catalog with direct integration into the GUI, no CLI needed. Direct support is available on the [TrueCharts Discord](https://discord.gg/tVsPTHWTtr) and the source code is available on [GitHub](https://github.com/truecharts/charts).
+Jellyfin is available as a [TrueNAS SCALE](https://www.truenas.org/) App in the official app catalog with direct integration into the GUI, no CLI needed.
 
-1. Install the TrueCharts Catalog to TrueNAS SCALE, see [website](https://truecharts.org/manual/SCALE/guides/getting-started/#adding-truecharts) for more info.
+1. Go to Apps page from the top level SCALE menu
 
-   1. Go to Apps page from the top level SCALE menu
-   2. Select Manage Catalogs tab on the Apps page
-   3. Click Add Catalog
-   4. After reading the iXsystems notice, click Continue and enter the required information:
-      Name: truecharts
-      Repository: `https://github.com/truecharts/catalog`
-      Preferred Trains: `enterprise` and `stable`
-      Branch: main
-   5. Click Save and allow SCALE to refresh its catalog with TrueCharts (this may take a few minutes)
+2. Click `Discover Apps` and search for `Jellyfin`
 
-2. Click `Available Applications` and search for `Jellyfin`
+3. Click `Install`, which will take you to the GUI installation wizard and you'll be able to fill out the necessary info
 
-3. Click `Install`, which will take you to the GUI Wizard and you'll be able to fill out the necessary info
+   - Published Server URL to publish in UDP Auto Discovery response
+   - User and Group Configuration, Network Configuration
+   - Storage Configuraton
+     - For Jellyfin Config Storage and Jellyfin Cache Storage, consider mapping Host Paths for ease of backup
+     - For Jellyfin Transcode Storage, consider using `Temporary` or `tmpfs`
+     - For media, add your library directories as Additional Storage
 
```

Comment:
Can you have instructions on how to give Jellyfin access to GPUs? Also disclaim that users might need to check kernel version on newer Intel GPUs

Reply:
Good point, completely forgot about the GPUs there. They actually have option at the very bottom Resources Configuration -> GPU Configuration -> Passthrough available (non-NVIDIA) GPUs. This leaves the open question of what about NVIDIA GPUs xD

Reply:
> Also disclaim that users might need to check kernel version on newer Intel GPUs

Current and upcoming versions of TrueNAS (24.10 and 24.04) seem to use 6.6, while previous old 23.10 used 6.1. I read that Intel Arcs were introduced in 6.2. So in a week it'll be last two stable versions already supporting them, with which I think it might be fine to shorten the docs by not mentioning that maybe if it's new enough?

I mean TrueNAS has a read-only main system fs, so it would be somewhat challenging to upgrade the kernel to begin with too :/ so the mention should boil down to whether 6.6 is considered new enough to not cause any problems

Reply:
Transcoding mention done. For the record, this is how the screen is supposed to appear there:
![](https://www.truenas.com/docs/images/SCALE/Apps/InstallCustomAppAddResourceLimits.png)

I didn't go into more details on how to configure it within Jellyfin, as the instructions for Docker etc above don't go either. I don't remember if the `/dev/dri/renderD128` is configured in Jellyfin by default, but in my case having VA-API and that path in the settings works then.

Reply:
Some Intel GPUs need 6.7 so a disclaimer is still needed. A short disclaimer linking the Intel known issues and limitations page under hardware acceleration, and suggest them to use a VM with pcie pass through would be sufficient